src/pcappy/__init__.py

```diff
@@ -276,6 +276,20 @@ def netmask(self):
 
     mask = netmask
 
+    def match(self, packet, length=None, caplen=None):
+        pkt = str(packet)
+        hdr = pcap_pkthdr()
+        if length is None:
+            length = len(pkt)
+        if caplen is None:
+            caplen = length
+        hdr.len = length
+        hdr.caplen = caplen
+        return pcap_offline_filter(pointer(self._bpf), pointer(hdr), pkt)
```

Comment:
This should probably return:
pcap_offline_filter(pointer(self._bpf), pointer(hdr), pkt) != 0